lib/simulator-xcode-6.js

```diff
@@ -185,6 +185,34 @@ class SimulatorXcode6 extends EventEmitter {
     return appDirs.length !== 0;
   }
 
+  /**
+   * Returns user installed bundle ids which has 'bundleName' in their Info.Plist as 'CFBundleName'
+   * @param {string} bundleId - The bundle id of the application to be checked.
+   * @return {array<string>} - The list of bundle ids which have 'bundleName'
+   */
+  async getUserInstalledBundleIdsByBundleName (bundleName) {
+    const rootUserAppDir = await this.buildBundlePathMap('Bundle');
+    const bundleIds = [];
+    if (_.isEmpty(rootUserAppDir)) {
+      return bundleIds;
+    }
+
+    for (const [bundleId, userAppDirPath] of Object.entries(rootUserAppDir)) {
+      const appFile = (await fs.readdir(userAppDirPath)).find(
+        (file) => path.extname(file).toLowerCase() === '.app');
```

Comment:
I personally find it more readable if the starting point is moved to the next line, for example

```
const r = arr.map(...)
  .filter(...)
  .find(...)
```